sdcm/utils/replication_strategy_utils.py

```diff
@@ -48,17 +57,21 @@ class NetworkTopologyReplicationStrategy(ReplicationStrategy):
 
     def __init__(self, default_rf: int | None = None, **replication_factors: int):
         if default_rf is not None:
-            self.replication_factors = {"replication_factor": default_rf}
+            self._replication_factors = {"replication_factor": default_rf}
         else:
-            self.replication_factors = {}
-        self.replication_factors.update(**replication_factors)
-        if not self.replication_factors:
+            self._replication_factors = {}
+        self._replication_factors.update(**replication_factors)
+        if not self._replication_factors:
```

Comment:
uh, I didn't notice we already took `replication_factors`. I wouldn't change it to protected (especially when it's used outside now). Also replication_factors shows datacenters, while new property only factors. So, I propose to rename `_replication_facors` to `replication_factors_per_dc` or `datacenter_replication_factors` - or any other you feel fits better.
Then pylint protected access disable can be removed.

Reply:
renamed